public/i18n/en.json

```diff
@@ -168,5 +168,9 @@
   "ftm-thanks-smartling": "Smartling for access to their Translation Managment Platform",
   "ftm-thanks-google": "Google for Maps API credits",
   "ftm-thanks-slack": "Slack for a Pro account",
-  "ftm-tweet-share-button": "Find the closest place to donate masks and other PPE at www.findthemasks.com"
+  "ftm-tweet-share-button": "Find the closest place to donate masks and other PPE at www.findthemasks.com",
+  "ftm-nav-about": "About",
+  "ftm-nav-partners": "Partners",
+  "ftm-nav-volunteer": "Volunteer",
+  "ftm-nav-addplace": "Add a place"
```

Comment:
Should this be "Site"? I think that's the language we currently use

Reply:
@drwestco we settled with "I need PPE" CC: @tinabeans